src/main/java/core/basesyntax/service/impl/DataParserImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+    private static final String SEPARATOR = ",";
+    private static final int INFORMATION_LINE_INDEX = 0;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        lines.remove(INFORMATION_LINE_INDEX);
+        List<FruitTransaction> transactions = new ArrayList<>();
+        lines = removeSpaces(lines);

```

Comment:
The method 'removeSpaces' is called on the 'lines' list but it also modifies the 'lines' list internally which can be confusing. It would be better to perform this operation without reassigning 'lines' to keep the code clean and understandable.